content/api/v18/async_hooks.en.md

```diff
@@ -449,22 +449,22 @@ async_hooks.createHook({
     const indentStr = ' '.repeat(indent);
     fs.writeSync(
       fd,
-      `$indentStr$type($asyncId):` +
-      ` trigger: $triggerAsyncId execution: $eid\n`);
+      `${indentStr}${type}(${asyncId}):` +
+      ` trigger: ${triggerAsyncId} execution: ${eid}\n`);
   },
   before(asyncId) {
     const indentStr = ' '.repeat(indent);
-    fs.writeSync(fd, `$indentStrbefore:  $asyncId\n`);
+    fs.writeSync(fd, `${indentStr}before:  ${asyncId}\n`);
     indent += 2;
   },
   after(asyncId) {
     indent -= 2;
     const indentStr = ' '.repeat(indent);
-    fs.writeSync(fd, `$indentStrafter:  $asyncId\n`);
+    fs.writeSync(fd, `${indentStr}after:  ${asyncId}\n`);
```

Comment:
Also looks off

Reply:
Actuall not. https://raw.githubusercontent.com/nodejs/node/main/doc/api/async_hooks.md

Reply:
Fair enough. What I was getting at is if you do a clean 18.9.0 sync before these changes, it's easier to see what the actual linting changes are introducing

Reply:
Gotcha, I'm going to decouple the API fixes and API updates from this PR. Hold tight

Reply:
https://github.com/nodejs/nodejs.dev/pull/2758